front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationDescription.tsx

```diff
@@ -0,0 +1,174 @@
+import { Fab, Grid, Paper, TextField, Typography } from "@material-ui/core";
+import ClearIcon from "@material-ui/icons/Clear";
+import DoneIcon from "@material-ui/icons/Done";
+import EditIcon from "@material-ui/icons/Edit";
+import React, { Fragment } from "react";
+import { updateOrganizationDescriptionDatabase } from "../../../Scripts/firebaseOrganizationUpdates";
+import { PageProps } from "../../Pages";
+
+declare interface ViewEditOrganizationInfoProps {
+  currentOrganizationData: any;
+  setNotification: any;
+  handleLoadOrganizationData: any;
+  setLoadingMessage: any;
+  classes: any;
+}
+
+const ViewEditOrganizationDescription: React.FunctionComponent<ViewEditOrganizationInfoProps> = ({
+  currentOrganizationData,
+  setNotification,
+  handleLoadOrganizationData,
+  setLoadingMessage,
+  classes,
+}) => {
+  const [organizationDescription, setOrganizationDescription] = React.useState<
+    string
+  >(
+    currentOrganizationData?.organizationDescription
+      ? currentOrganizationData.organizationDescription
+      : ""
+  );
+  const [editing, setEditing] = React.useState(!organizationDescription);
+
+  const startEditing = () => {
+    setEditing(true);
+  };
+
+  const cancelEditing = () => {
+    setEditing(false);
+  };
+
+  const cancelEditingOrganizationDescription = () => {
+    cancelEditing();
+    setOrganizationDescription(
+      currentOrganizationData?.organizationDescription
+        ? currentOrganizationData.organizationDescription
+        : ""
+    );
+  };
+
+  const saveOrganizationDescription = () => {
+    if (currentOrganizationData) {
+      setLoadingMessage("Updating Organization Description...");
+      updateOrganizationDescriptionDatabase(
+        currentOrganizationData.userId,
+        organizationDescription
+      )
+        .then((value: any) => {
```

Comment:
You shouldn't need to specify any... See if you can take the any out.

Reply:
Done